### PR TITLE
podman-compose instead of docker-compose for Podman

### PR DESCRIPTION
**Pull request by ozkanonur**
Applied same containerization techniques that have been used for Anitya

### CODE REVIEW

File: docs/dev-guide.rst [hunk 1]
```diff
@@ -115,10 +113,9 @@ Hotness container starts after the start of containers running services required
 Makefile scripts that provide easier container management:
 
 * ``make up`` Starts all the container services
-* ``make restart`` Restarts all the container services that are either stopped or running 
+* ``make restart`` Restarts all the container services that are either stopped or running
 * ``make halt`` Stops and removes the containers
 * ``make bash`` Connects to hotness container
-* ``make logs`` Shows all logs of all containers
```
Comment (Zlopez) [inline docs/dev-guide.rst:121]: Why is this removed?

Comment (ozkanonur) [reply docs/dev-guide.rst:121]: Because logs command behaves differently on podman compose. You need to specify 1 container to show logs.

But I think I can do some workarounds. I will provide this command back.

**Comment by ozkanonur** [comment]
@Zlopez  May I know the state of this PR ?

**Comment by Zlopez** [comment]
Let me quickly test it, I forgot about this PR.

**Review by Zlopez — APPROVED** [review]
I tested it and everything seems to be working. Thanks for the work
